zerver/webhooks/slack/doc.html

```diff
@@ -0,0 +1,34 @@
+<p>Get notifications from Slack for messages on your team's public channels</p>
+<p>
+    There are two ways in which you may want to receive a notification
```

Comment:
I'm sorry that I didn't catch this earlier, but you should have a colon at the end of this line, since you're about to give the reader a list.